WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewModel.swift

```diff
@@ -216,6 +228,13 @@ extension IssueRefundViewModel {
         let shippingTotalUseCase = RefundShippingCalculationUseCase(shippingLine: shippingLine, currencyFormatter: formatter)
         return productsTotalUseCase.calculateRefundValues().total + shippingTotalUseCase.calculateRefundValue()
     }
+
+    /// Returns a string with the count of how many items are selected for refund.
+    ///
+    private func createSelectedItemsCount() -> String {
+        let count = state.refundQuantityStore.count()
+        return String.pluralize(count, singular: Localization.itemSingular, plural: Localization.itemsPlural)
+    }
```

Comment:
Another way is to move this to `Localization` instead since it's a logic that we shouldn't really need to care about if only we supported Strings dict. 

```swift
enum Localization {
    static func selectedItemsCountLabel(_ count: Int) -> String {
        String.pluralize(count, singular: Localization.itemSingular, plural: Localization.itemsPlural)
    }
}
```